engine-dmn/feel-juel/pom.xml

```diff
@@ -15,6 +15,7 @@
     <!-- We shade artifacts into the jar, so we need to generate 
     a dependency BOM for the license book -->
     <skip-third-party-bom>false</skip-third-party-bom>
+    <camunda.artifact>org.camunda.bpm.dmn.feel.impl.juel</camunda.artifact>
```

Comment:
Since this property is not referenced anywhere yet, I am not entirely sure what you are intending to do with it. However, please make sure this property is not colliding with the current definition of the bundle plugin further down in this file as we do specifically exclude some `*.impl.*` packages from the export directive.

Reply:
Hello @tmetzke ,

If you refer to this code https://github.com/camunda/camunda-bpm-platform/blob/master/parent/pom.xml#L81, the code is commented, and all packages are exported.
`<camunda.osgi.export.pkg>${camunda.artifact}*</camunda.osgi.export.pkg>`